tests/benchmark/Tracing/infection.json5

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
the config is now here in the benchmark rather than the donwstream package. Maybe that will need to change, but for now I think it's fine as it is